pika/connection.py

```diff
@@ -2176,12 +2187,12 @@ def _rpc(self, channel_number, method,
             raise TypeError('acceptable_replies should be list or None')
 
         # Validate the callback is callable
-        if callback_method:
-            if not utils.is_callable(callback_method):
+        if callback:
```

Comment:
Even if it's callable, the callback could be some strange user-supplied callable object, where the user overrode `__bool__` to do something funky, so `if callback` could evaluate to false even when callback is not None. The only safe thing to do is `if callback is not None:`.